src/test/java/com/epam/tc/hw2/ex1/LoginPageTest.java

```diff
@@ -0,0 +1,97 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.AbstractBaseTest;
+import java.util.ArrayList;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class LoginPageTest extends AbstractBaseTest {
+    SoftAssertions softly = new SoftAssertions();
+    static final String EPAM_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+
+    @Test
+    public void loginCheckTest() {
+        webDriver.navigate().to(EPAM_URL);
+        webDriverWait.until(ExpectedConditions.urlToBe(EPAM_URL));
+        //Assert Browser title
+        webDriverWait.until(ExpectedConditions.titleIs("Home Page"));
+        softly.assertThat(webDriver.getTitle()).isEqualTo("Home Page");
+        //Assert performing login
+        WebElement loginForm = webDriver.findElement(By.id("user-icon"));
+        loginForm.click();
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(loginForm));
+        WebElement username = webDriver.findElement(By.id("name"));
+        username.sendKeys("Roman");
+        webDriver.findElement(By.id("password"));
+        WebElement password = webDriver.findElement(By.id("password"));
+        password.sendKeys("Jdi1234");
+        WebElement loginButton = webDriver.findElement(By.id("login-button"));
+        loginButton.click();
+        webDriverWait.until(ExpectedConditions.urlToBe(EPAM_URL));
+        softly.assertThat(webDriver.getCurrentUrl()).isEqualTo(EPAM_URL);
+        // Check loggined
+        WebElement userNameLogged = webDriver.findElement(By.id("user-name"));
+        userNameLogged.isDisplayed();
```

Comment:
What do you try to check here? This code does not check anything

Reply:
I do not see any changes here. What do you try to check here?
Method `isDispayed()` return **true** if element is displayed on the page, otherwise **false**. But this method does not contain any verification of the displaying element